SWIGGYDEMO_REACT/swiggy-react-app/src/components/itemDetailsAndCart/components/cartItems/components/checkoutButton/CheckoutButtonContainer.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import CheckoutButton from "./CheckoutButton";
+
+function CheckoutButtonContainer ({onCheckout}) {
+    return (
+        <CheckoutButton data="CHECKOUT" onCheckout={() => onCheckout()}/>
```

Comment:
pass onCheckout directly no need to wrap within another arrow function.

Reply:
Passed onCheckout directly